linkml/generators/graphqlgen.py

```diff
@@ -19,6 +20,8 @@ class GraphqlGenerator(Generator):
     uses_schemaloader = True
     requires_metamodel = False
 
+    fail_on_missing_permissible_values = False
```

Comment:
Can we get a docstring for this?

Reply:
What do you mean? I cannot put a docstring on an assignation. Do you mean a comment? Or some further information at the class' docstring?

Reply:
sphinx supports docstrings on variable assignments :)

```python
variable = 'value'
"""
this is a docstring on a variable assignment :)
"""

#: this is a doc comment on a variable assignment
variable = 'value'
```

or yes, the class docstring could have

"""
Attributes:
    variable (bool): description
"""

Reply:
The former is more in line with existing practice in the generators and the repo. But also note that these may get *duplicate* definitions if the option is exposed via click. I think we should sort out best practice here outside this individual PR rather than holding it to inconsistent standards.

Reply:
Yes yes basically I just wanted some note somewhere about what this param does, there are a number of params without documentation so you're right I dont want to hold this up unfairly but it would be nice to have docs and ideally tests for all added features, but dont want to be exhausting

Reply:
Code documentation has been improved later. Please make suggestions to the code wherever more documentation desired.